app/src/main/res/values/strings.xml

```diff
@@ -62,21 +62,21 @@
     <string name="contents">Contents</string>
     <string name="qr">QR Code</string>
     <string name="cam_pm">Camera permission</string>
-    <string name="grant_pm_rationale">Please grant camera permission.\nThis is for this app to scan codes(barcode/QR code).</string>
-    <string name="pm_denial">Camera permission has been denied.</string>
-    <string name="pm_denial_permanant">Camera permission has been denied permanently. Please go to system settings.</string>
-    <string name="position_code">Position code in viewfinder</string>
+    <string name="grant_pm_rationale">Please grant camera permission to scan QR and barcodes.</string>
+    <string name="pm_denial">Camera permission denied.</string>
+    <string name="pm_denial_permanant">Camera permission denied permanently. Correct this in the system settings.</string>
+    <string name="position_code">Position the code in the square</string>
     <string name="codes_scanned">%1$d codes scanned</string>
     <string name="date_icon">Date icon</string>
-    <string name="data_type_code">Data type of code scanned</string>
-    <string name="format_code">Format of code scanned</string>
-    <string name="qr_color">Color</string>
-    <string name="background_color">Background color</string>
+    <string name="data_type_code">Datatype of the scanned code</string>
+    <string name="format_code">Format of the scanned code</string>
+    <string name="qr_color">Colour</string>
```

Comment:
The source language is en_UK in https://hosted.weblate.org/projects/code-scanner/code-scanner-main/ (en_US) can be added where those strings are populated.